constructors.js

```diff
@@ -71,8 +74,14 @@ console.assert(new Dog().says === 'life is ruff', 'all Dogs should have a method
 // create an instance of this called `cat`
 
 // -- ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ - Your Answer - ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ -- //
+function Cat() {}
 
+Cat.prototype.growl = function() {
+  return 'meow';
+};
 
+var cat = new Cat();
+cat.growl();
```

Comment:
You don't need to actually call `cat.growl` that is tested on line 91 in `console.assert`